contracts/bitizens/AvatarItemToken.sol

```diff
@@ -6,218 +6,280 @@ import "./AvatarItemService.sol";
 
 contract AvatarItemToken is ERC721ExtendToken, AvatarItemService, AvatarChildService {
 
-  event TokenBurned(address owner, uint256 tokenId, uint256 _totalBurned);
-
-  struct AvatarItem {
-    // the listed number, the number is from the content master list
-    uint32 listNumber;
-    // set number, max is 99      
-    uint16 setNumber;
-    // true  => this itme is for avatar
-    // false => this item not for avatar
-    bool isBitizenItem;
-    // flase => This item is not part of a set    
-    // true => This item is a part of a set
-    bool isSet; 
-    // 00 => clothes  
-    // 01 => pet 
-    // type may be add in the future
-    uint8 types;
-    //01 => Common
-    //02 => Uncommon  
-    //03 => Rare      
-    //04 => Epic 
-    //05 => Legendary 
-    //06 => Godlike   
-    //07 => Unique
-    uint8 rarity;  
-    // 01 => Head
-    // 02 => Top
-    // 03 => Bottom
-    // 04 => Feet           
-    // 05 => Trinket      
-    // 06 => Acc  
-    // ...=> more 
-    uint8 socket;
-    // 00 => Male     
-    // 01 => Female     
-    // 10 => Male-only      
-    // 11 => Female-only
-    uint8 gender;
-    // extProps for future 
-    uint8 ext1;
-    // extProps for future
-    uint8 ext2;
-  }
-
-  AvatarItem[] avatarItems;
-
-  mapping(uint256 => AvatarItem) burnTokens;
-
-  uint256[] burnTokenIds;
-
-  mapping(uint256 => bool) isBurnByTokenId;
+  struct Item {
+    address foundedBy;    // item founder
+    address createdBy;    // item creator
+    bool isBitizenItem;   // true for avatar false for other
```

Comment:
very clear comments here, 👍 
if this is checking for Avatar, will it be clearer to just name it isAvatar?
isBitizenItem makes me think it should be true for an item, but false for avatar (the opposite)
Keeping as-is is fine with me, too.

Reply:
My fault. It is real means  for bitizen,I'll fix it